Update partitioned-table.md (#16598)
pingcap/docs: partitioned-table.md

PARTITIONS 4;
```

> TiDB also supports creating Key partitioned tables with an empty partition column list specified in `PARTITION BY KEY`, just like MySQL. For example, the following statement will create a partitioned table with the primary key `id` as the partitioning key:

Reviewer: ```suggestion
Similar to MySQL, TiDB supports creating Key partitioned tables with an empty partition column list specified in `PARTITION BY KEY`. For example, the following statement creates a partitioned table using the primary key `id` as the partitioning key:
```

PARTITIONS 4;
```

> If there is no primary key but a unique key, the unique key will be used as the partitioning key:

Reviewer: ```suggestion
If the table lacks a primary key but contains a unique key, the unique key is used as the partitioning key:
```

> ```

Reviewer: ```suggestion
```sql
```
